packages/material-ui/src/BottomNavigationAction/BottomNavigationAction.js

```diff
@@ -56,57 +55,56 @@ export const styles = theme => ({
   },
 });
 
-class BottomNavigationAction extends React.Component {
-  handleChange = event => {
-    const { onChange, value, onClick } = this.props;
+const BottomNavigationAction = React.forwardRef(function BottomNavigationAction(props, ref) {
+  const {
+    classes,
+    className: classNameProp,
```

Comment:
```suggestion
    className,
```

Reply:
I have removed the prop rename and the intermediary `className` variable in a lot of components. I find it simpler without these two moving parts. What do you think?

Reply:
Sure, when the `className` logic isn't too complex. It get's a bit unwieldy having larger class definitions inline.